Tests/TuistKitTests/ImportFinder/ImportSourceCodeScannerTests.swift

```diff
@@ -0,0 +1,173 @@
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistKit
+
+final class ImportSourceCodeScannerTests: TuistUnitTestCase {
+    func test_whenObjcCodeWithImports() throws {
+        let code = """
+        #import <UIKit/UIKit.h>
+        #import <A/SomeHeader.h>
+
+        @interface UYLAppDelegate : UIResponder <UIApplicationDelegate>
+
+        @property (strong, nonatomic) UIWindow *window;
+
+        @end
+        """
+        let imports = try ImportSourceCodeScanner().extractImports(from: code, language: .objc)
```

Comment:
I'd create a `subject` property for `ImportSourceCodeScanner` as we do in other unit tests